package.json

```diff
@@ -6,7 +6,7 @@
     "export-tests": true
   },
   "scripts": {
-    "pretest": "test -z \"$JENKINS_HOME\" || (echo | node bin/setup-mysql.js)",
```

Comment:
We can try it and find out. The alternative is to modify `bin/setup-mysql.js` so that it doesn't use `read()`.